File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int se = 0, ga = 0;
        if(sizes[0][0] > sizes[0][1]){
            se =sizes[0][0]; ga = sizes[0][1];
        } else{
            se =sizes[0][1]; ga = sizes[0][0];
        }
        for(int i = 0; i < sizes.length; i++){
            if(sizes[i][0] > sizes[i][1]){
                se = sizes[i][0] > se ? sizes[i][0] : se;
                ga = sizes[i][1] > ga ? sizes[i][1] : ga;
            } else {
                se = sizes[i][1] > se ? sizes[i][1] : se;
                ga = sizes[i][0] > ga ? sizes[i][0] : ga;
            }
        }
        return se*ga;
    }
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = [0,0]
    for i in range(len(sizes)):
        sizes[i].sort()
        if sizes[i][0] > answer[0]:
            answer[0] = sizes[i][0]
        if sizes[i][1] > answer[1]:
            answer[1] = sizes[i][1]
    return answer[0] * answer[1]